### PR TITLE
Fixes #2263, #2264: Removes links and icons from print

### LINKED ISSUE #2263: Printing a test run adds entity paths
### Description of problem

When printing a test run, KIWI TCMS adds entity paths in brackets after the entity names. This creates a bloated report that's hard to read.

### Version or commit hash (if applicable)

9.0.1-Enterprise

### Steps to Reproduce

1. Open a test run (e.g. https://public.tenant.kiwitcms.org/runs/2861/)
2. Hit CMD+P to print.

### Actual results

"Produkt: Kiwi TCMS (/runs /search/?product=2660)"
"Version: unspecified (/runs /search/?product=2660& product_version=3562)"
"Build: 2020-12-14 (/runs /search/?product=2660& product_version=3562& build=2882)"

### Expected results

"Produkt: Kiwi TCMS"
"Version: unspecified"
"Build: 2020-12-14"

### Additional info (Python traceback, logs, etc.)

I'm using macOS 11.1 Big Sur and Firefox 85.0, german locale.

### LINKED ISSUE #2264: Printing a test run adds buttons
### Description of problem

When printing a test run, the buttons to add users, tags or attachments are printed as well. This creates a bloated report that looks unprofessional.

### Version or commit hash (if applicable)

9.0.1-Enterprise

### Steps to Reproduce

1. Open a test run (e.g. https://public.tenant.kiwitcms.org/runs/2861/)
2. Hit CMD+P to print.

### Actual results

The upload buttons are printed:

```
Befestigung
Datei Besitzer Datum
Keine Belege gefunden
Upload attachment:
Durchsuchen... Keine Datei ausgewählt.
Add attachment
```
### Expected results

Upload buttons should be omitted.

```
Befestigung
Datei Besitzer Datum
Keine Belege gefunden
```

### Additional info (Python traceback, logs, etc.)

I'm using macOS 11.1 Big Sur and Firefox 85.0, german locale.

### PR DESCRIPTION
**Pull request by pandafy**
Closes #2263 
Closes #2264 

@atodorov this is a draft implementation(but pretty much all needed), what do you think about it? Would be good to generalise it such that it applies to `get` views of other models(Test Plan, Test Case) as well? If so, we will have to add conditions for some corner cases. 

Also, will you be able to share some hints on how to write tests for this?

### CODE REVIEW

**Comment by pandafy** [comment]
@dkoerner85 does this look good to you? 

image

File: tcms/static/style/patternfly_override.css [hunk 1]
```diff
@@ -39,3 +39,15 @@ nav.navbar-default { border-top-color: #3C8D2C; }
...
+@media print {
+    a[href]:after,
+    form.toolbar-pf-actions,
+    .card-pf-accented tfoot,
+    .list-view-pf-actions,
+    .list-view-pf-checkbox,
+    .pficon-error-circle-o,
+    .tree-list-view-pf .comment-form {
+        display: none !important;
+    }
```
Comment (atodorov) [inline tcms/static/style/patternfly_override.css:52]: Patternfly already has the `.hidden-print` class definition and we'd like to use that as much as possible.

Also we want to have elements in  HTML templates explicitly marked with this class to that it is clear they are not intended to be printed on the screen. 

An extension for the `:after` elements showing the URLs will be needed probably.

**Review by atodorov — CHANGES_REQUESTED** [review]

**Comment by dkoerner85** [comment]
> @dkoerner85 does this look good to you?
> image

That looks pretty clean to me!

File: tcms/testruns/templates/testruns/get.html [hunk 2]
```diff
@@ -452,7 +452,7 @@ <h5><span class="test-executions-count"></span> {% trans 'records' %}</h5>
 
                                             </div>
                                         </div>
-                                        <div class="col-md-6">
+                                        <div class="col-md-6 hidden-print">
```
Comment (atodorov) [inline tcms/testruns/templates/testruns/get.html:455]: This hides all existing comments and attachments as well, not only the form.

**Review by atodorov — CHANGES_REQUESTED** [review]
Minor changes requested. 

info icon for bugs & hyperlinks doesn't need to be printed as well.

**Review by atodorov — CHANGES_REQUESTED** [review]
Status progressbar colors are not printed, only the numbers representing percentages are shown. When you have pass/fail combination this isn't very intuitive b/c everything is the same color. IMO don't print the progressbar at all, leave only the table with statuses.

**Comment by atodorov** [comment]
> info icon for bugs & hyperlinks doesn't need to be printed as well.

Still unresolved.

**Comment by pandafy** [comment]
@atodorov can you please point which icons you are referring to? It will be very helpful.

**Comment by atodorov** [comment]
bugs_and_hyperlink_icon

**Comment by pandafy** [comment]
Oops! I hid the wrong element before. Thanks Alexander! :raised_hands:
